lisp/old/ess-old-setup.el

```diff
@@ -0,0 +1,344 @@
+;;; ess-old-setup.el --- Old setup suggestions from ess-site.ely
+
```

Comment:
> I think setup is more suggestive.

Maybe but it's not conventional. ESS users are used to -site. Emacs and other *nix packages are using -site for site-wise configuration. Emacs packages are using `-config.el` for examples of configuration.

Let's just keep it ess-site-old.el. Such a name would quickly pop up in google search is old users would search for ess-site.